packages/braid-design-system/src/lib/components/Hidden/Hidden.tsx

```diff
@@ -32,18 +33,25 @@ export const Hidden = ({
   data,
   ...restProps
 }: HiddenProps) => {
-  if (process.env.NODE_ENV === 'development' && screen) {
-    // eslint-disable-next-line no-console
-    console.warn(
-      `You used the "screen" prop on Hidden, but this probably doesn't do what you expect. If you're trying to provide content to screen readers without rendering it to the screen, use the <HiddenVisually> component instead. The "screen" prop is likely to be deprecated in a future release.`,
-    );
+  if (process.env.NODE_ENV === 'development') {
+    if (screen) {
+      // eslint-disable-next-line no-console
+      console.warn(
+        `You used the "screen" prop on Hidden, but this probably doesn't do what you expect.
+        If you're trying to provide content to screen readers without rendering it to the screen, use the <HiddenVisually> component instead.
+        The "screen" prop is deprecated and will be removed in a future release.`,
```

Comment:
The message could probably do with some better guidance than what we had before.

Thoughts on something like this:

> The `screen` prop is deprecated and will be remove in a future release.
> For content designed to improve the screen reader experience, please use <HiddenVisually> instead.
> Alternatively, if the content should be hidden unconditionally, remove the `screen` prop.